src/components/tab-bar/tab-bar.scss

```diff
@@ -25,6 +25,46 @@ $tab-separator-background-color: #e8e8ea;
     }
 }
 
+.mdc-tab-scroller {
+    position: relative;
+    overflow: hidden;
+
+    &::before,
+    &::after {
+        content: " ";
+        display: block;
+        position: absolute;
+        top: 0;
+        height: 100%;
+        width: 10rem;
+        z-index: 1;
+        pointer-events: none;
+        transition-property: transform;
+        transition-duration: 250ms;
+        transition-timing-function: ease-out;
+    }
+
+    &::before {
+        transform: translate3d(-10rem, 0, 0);
+        left: 0;
+        background: linear-gradient(270deg, rgba(246,246,247,0) 0%, rgba(246,246,247,.9) 80%, rgba(246,246,247,.9) 100%);
```

Comment:
> Sometimes it's hard to see/understand that there's more content, maybe some less fading will do it

That's always a bit of a problem when using fading. Using buttons with arrows instead, like @jensgustafsson suggested, ought to solve that problem though.

Reply:
I think we should try buttons. I think the fading looks really great but unfortunately that doesn't make me understand that there is more tabs if I scroll

Reply:
I'm working it now 🙂

@Kiarokh posted this design on the Issue, which I'm going to see if I can pull off:

![image](https://user-images.githubusercontent.com/35954987/79472885-ac782180-8004-11ea-9e7b-ecd1a4940ce9.png)

Reply:
Now there are buttons. Not yet quite as nice looking as those in the mock, but functional at least 🙂